### ISSUE TITLE
support for adding modules with subpath export

### ISSUE THREAD

**Issue by danielroe** [nuxt/cli#218]
e.g. https://louismazel.github.io/maz-ui-3/guide/nuxt

(related: https://github.com/nuxt/modules/pull/773/files)

**Comment by pi0** [comment]
We might auto detect subpath and also allow `nuxi module add mazui/nuxt` (which is not short) but wondering maybe would be good idea to support a field in modules DB as well?

**Comment by danielroe** [comment]
Exactly, I think it would be good to have something there that's the module to add to the modules array, if it's a subpath.
